_gsocorgs/2020/cross.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Center for Research in Open Source Software"
+author: "Jeff LeFevre"
+layout: default
+organization: CROSS
+logo: CROSS-logo.png
+description: |
+  The Center for Research in Open Source Software (www.cross.ucsc.edu), part of the Baskin School of Engineering at UC Santa Cruz, was founded based on the success of former UCSC student Sage Weil and the Ceph storage system.  At CROSS, we create open source leaders by blending open source software strategies with graduate-level research. CROSS turns OSS prototypes into successful open source software projects.
```

Comment:
It seems me a link would be more readable…!  [Center for Research in Open Source Software](www.cross.ucsc.edu)